draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1961,17 +2026,18 @@ Operations:
 2. tv2 = 1 + tv1
 3. tv1 = 1 - tv1
 4. tv3 = inv0(tv1 * tv2)
-5. tv4 = sqrt(-g(Z) * (3 * Z^2 + 4 * A))
+5. tv4 = sqrt(-g(Z) * (3 * Z^2 + 4 * A))    # can be precomputed
 6. If sgn0(tv4) == 1, set tv4 = -tv4        # sgn0(tv4) MUST equal 0
 7. tv5 = u * tv1 * tv3 * tv4
-8.  x1 = -Z / 2 - tv5
-9.  x2 = -Z / 2 + tv5
-10. x3 = Z - 4 * g(Z) * (tv2^2 * tv3)^2 / (3 * Z^2 + 4 * A)
-11. If is_square(g(x1)), set x = x1 and y = sqrt(g(x1))
-12. Else If is_square(g(x2)), set x = x2 and y = sqrt(g(x2))
-13. Else set x = x3 and y = sqrt(g(x3))
-14. If sgn0(u) != sgn0(y), set y = -y
-15. return (x, y)
+8. tv4 = -4 * g(Z) / (3 * Z^2 + 4 * A)      # can be precomputed
```

Comment:
Should this be `tv6`?

Reply:
Sure, that might be clearer. I'll make the change :+1: